db/migrate/20220217104803_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Post do |t|
+      t.references :User, null: false, foriegn_key: true, index:true
+      t.string :Title
+      t.string :Text
+      t.timestamps
+      t.integer :Comments_Counter ,default: 0,null: false
+      t.integer :Likes_Counter ,default: 0,null: false
```

Comment:
- [ ] <b>REQUIRED:</b> In rails, it is much better to use `lower case` and `snake case` when naming. For example in this case, `Title` should be `title`,  `Text` should be `text`,  `Comments_Counter ` should be `comments_counter `,  and `Likes_Counter ` should be `likes_counter `.